gsy_framework/schema/avro_schemas/scm_simulation_raw_data_v1.json

```diff
@@ -0,0 +1,199 @@
+{
```

Comment:
Can you please explain why you use the `_v1` suffix? Are we planning to have multiple versions?
IMO, if the data structure changes, we need to change the schema, too.

Reply:
According to my research about schema management, different services of a system may be using different versions of the same schema. I just added the _v1 suffix to conform to the standards.

Reply:
Hmm, I do not foresee us using different schema versions. And IMO it is confusing for the implementor of the validator to have to decide between different versions. There should only be one schema for the raw_data IMO. 
Do you agree?

Reply:
OK. I can remove the _v1 suffix for now. We are still able to use them in the future if required.

Reply:
Done.